src/NHSD.BuyingCatalogue.Identity.Api/Controllers/ConsentController.cs

```diff
@@ -56,10 +57,10 @@ public async Task<IActionResult> Index(ConsentViewModel model)
         [HttpGet("/dismiss-cookie-banner")]
         public IActionResult DismissCookieBanner()
         {
-            Response.Cookies.Append(Cookies.BuyingCatalogueConsent, "true", new CookieOptions
-            {
-                Expires = DateTime.Now.Add(cookieExpiration.ConsentExpiration),
-            });
+            Response.Cookies.Append(
+                Cookies.BuyingCatalogueConsent,
+                DateTime.Now.ToString("g", CultureInfo.CurrentCulture),
+                new CookieOptions { Expires = DateTime.Now.Add(cookieExpiration.ConsentExpiration), });
```

Comment:
We'll need  to make sure the content of the cookie is consistent across .NET and node code. The node code that creates the cookie looks like this (from the buying-catalogue-library):

```js
const millisecondsInOneYear = 31556926000;
  const cookieData = {
    cookieValue: true,
    creationDate: new Date().getTime(),
  };
  res.cookie(config.consentCookieName, `${JSON.stringify(cookieData)}`, { maxAge: millisecondsInOneYear });
```

Reply:
My bad, corrected, thanks.